src/autolabel/tasks/base.py

```diff
@@ -94,6 +91,22 @@ def _is_few_shot_mode(self) -> bool:
     def construct_prompt(self, input: str, examples: List) -> str:
         pass
 
+    def construct_refuel_prompt(self, input: str, examples: List) -> str:
+        self.use_refuel_prompt_schema = True
+        self.prompt_template = PromptTemplate(
+            input_variables=get_format_variables(
+                self.FEW_SHOT_TEMPLATE_REFUEL_LLM
+                if self._is_few_shot_mode()
+                else self.ZERO_SHOT_TEMPLATE_REFUEL_LLM
+            ),
+            template=self.example_template,
+        )
+        refuel_prompt = self.construct_prompt(input, examples)
```

Comment:
hmm, i suggest instead passing this flag as a parameter to ` self.construct_prompt(...)` to denote which "prompt modes" to construct - the default one or the one customized for refuel llm. that seems cleaner than setting some isntance variavle, and then resetting it again.